enes-homework-week3-moreJS.js

```diff
@@ -0,0 +1,200 @@
+//Q:More JS
+
+//Q1
+
+function sum(a,b,c) {
+  return (a+b+c);
```

Comment:
This is called as a pure function where the function works on the inputs provided to it as arguments i.e `a`, `b` and `c` and then returns a computed value based on those 3 arguments.